templates/csharp/app-mvvm/App.axaml.cs

```diff
@@ -23,7 +24,11 @@ public override void OnFrameworkInitializationCompleted()
 #if (CommunityToolkitChosen)
             // Line below is needed to remove Avalonia data validation.
             // Without this line you will get duplicate validations from both Avalonia and CT
+    #if (AvaloniaStableChosen)
+            ExpressionObserver.DataValidators.RemoveAll(x => x is DataAnnotationsValidationPlugin);
+    #else
             BindingPlugins.DataValidators.RemoveAt(0);
```

Comment:
Hello @Mrxx99 , You can't assume there is only one Plugin in `DataValidators`.
![image](https://user-images.githubusercontent.com/10996250/216907930-fd9bd350-31f1-4ce4-b3e8-679ea0c14b40.png)
I suggest using a loop to remove all plugins (Equivalent to what the stable version does). Something like:
```csharp
for(var plugin = 0; plugin < BindingPlugins.DataValidators.Count; plugin++)
{
     BindingPlugins.DataValidators.RemoveAt(plugin);
}
```